.circleci/config.yml

```diff
@@ -115,8 +113,37 @@ jobs:
             #tar -zcv -f nupic_core-${CIRCLE_SHA1}-darwin64.tar.gz dist
             #aws s3 cp nupic_core-${CIRCLE_SHA1}-darwin64.tar.gz s3://artifacts.numenta.org/numenta/nupic.core/circle/
 
+  # job: ARM64 via Docker + QEMU ("arm64-build-test")
+  arm64-build-test:
+    docker:
+      # CircleCI base Ubuntu 18.10 Cosmic with remote Docker access, etc.
+      #   https://circleci.com/docs/2.0/circleci-images/#buildpack-deps
+      - image: circleci/buildpack-deps:18.10
+    steps:
+      - run:
+          # This is for local test builds using the CircleCI terminal app.
+          #   Local Docker seems to use sudo, CircleCI Docker does not.
+          command: |
+            if [[ $CIRCLE_SHELL_ENV == *"localbuild"* ]]; then
+              echo "This is a local build. Enabling sudo for docker"
+              echo 'export CI_SUDO="sudo"' >> $BASH_ENV
+            else
+              echo 'export CI_SUDO=' >> $BASH_ENV
+            fi
+            source $BASH_ENV
+      - run: sudo apt-get -y install qemu-system
+      - checkout
+      - setup_remote_docker  # access circleci's custom docker service
+      # here we make circleci x86 hardware think it's arm64 via docker+qemu
+      - run: $CI_SUDO docker run --rm --privileged multiarch/qemu-user-static:register --reset
+      - run:
+          command: $CI_SUDO docker build --build-arg arch=arm64 .
+          no_output_timeout: 2h
```

Comment:
question, how long to the tests (on CI in docker...) take to run?

Reply:
@breznak arm64 is 1 hour 45 min, mac is 25 min.